source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -148,6 +148,10 @@
 
 /*---------------------------------------------------------------------------*/
 
+#if ( ( tskKERNEL_VERSION_MAJOR < 8 ) || ( ( tskKERNEL_VERSION_MAJOR == 8 ) && ( tskKERNEL_VERSION_MINOR < 1 ) ) )
```

Comment:
I suppose that this specific version is used due to the fact that before this `pdMS_TO_TICKS` did not exist which is used a lot in this library.

Reply:
It's actually from [this](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1084#issuecomment-1997748807) but looking at the changelog myself I think maybe it should be v9.0 cause that's when task notifications were added. I can try and at least see the oldest version that will compile later.

Reply:
Thank you! That would be great.

Reply:
@HTRamsey 

Task notifications are used by the IP stack only if `ipconfigSELECT_USES_NOTIFY` is enabled. Network interfaces might still use it, depending on how the EMAC task is implemented.